Source/CarthageKit/Git.swift

```diff
@@ -247,23 +247,17 @@ public func cloneSubmoduleInWorkingDirectory(submodule: Submodule, workingDirect
 				return SignalProducer(error: CarthageError.RepositoryCheckoutFailed(workingDirectoryURL: submoduleDirectoryURL, reason: "could not enumerate name of descendant at \(URL.path!)", underlyingError: error))
 			}
 
-			if let name = name as? NSString {
-				if name != ".git" {
-					return .empty
-				}
-			} else {
-				return .empty
-			}
-
+            if (name as? String) != ".git" {
+                return .empty
+            }
```

Comment:
@mdiep Sorry about that! My xvim settings got in the way here, I will be more careful next time!